typos

eelucio opened the pull request.

eelucio committed:
typos

antarcticrainforest commented on src/evaluation_system/api/parameters.py:
```diff
@@ -136,7 +136,7 @@ def synchronize(self, tool: str) -> int:
         ----------
         tool:
-            name of the plugin that is synchronised.
+            name of the plugin that is synchronized.
```
Let's stick to British spelling.

antarcticrainforest commented on src/evaluation_system/misc/exceptions.py:
```diff
@@ -38,7 +38,7 @@ class ConfigurationException(Exception):
 
 class PluginNotFoundError(Exception):
-    """Exeption Definition for missing Plugins."""
+    """Exception Definition for missing Plugins."""
```
Definition -> definition

antarcticrainforest reviewed (approved):
Thanks a lot for this effort. Two very minor comments only, after which you can merge this.

antarcticrainforest commented:
In line 32 of the _databrowser.py file could you also change the sentence?

`raise ValueError(f"Time select method has one of {methods}") from error`

to `raise ValueError(f"Time select method has to be one of {methods}") from error`

eelucio committed:
correction typos

eelucio commented:
thanks!

antarcticrainforest commented:
Can you fix the linting errors?

eelucio commented:
When I do

```
$ make lint
mypy --install-types --non-interactive
Success: no issues found in 40 source files
black --check -t py310 src
All done! ✨ 🍰 ✨
84 files would be left unchanged.
```

I do not get linting errors for py310.

antarcticrainforest commented:
What happens if you update black in your environment?

eelucio commented:
same thing

```
$ python3 -m pip install -U black
...
    Uninstalling black-23.1.0:
      Successfully uninstalled black-23.1.0
Successfully installed black-23.7.0

$ make lint
mypy --install-types --non-interactive
Success: no issues found in 40 source files
black --check -t py310 src
All done! ✨ 🍰 ✨
84 files would be left unchanged.
```
the python I have is python 3.10.9

maybe I should reinstall the environment

antarcticrainforest commented:
With a python3.11 env?

eelucio commented:
I am currently updating the environemnt. As soon as it finishes I will try it again.

eelucio commented:
```
$ make lint
mypy --install-types --non-interactive
Success: no issues found in 40 source files
black --check -t py311 src
All done! ✨ 🍰 ✨
84 files would be left unchanged.
black --check -t py310 src
All done! ✨ 🍰 ✨
84 files would be left unchanged.
black --check -t py39 src
All done! ✨ 🍰 ✨
84 files would be left unchanged.
black --check -t py38 src
All done! ✨ 🍰 ✨
84 files would be left unchanged.
black --check -t py37 src
All done! ✨ 🍰 ✨
84 files would be left unchanged.
(freva-dev) [etor@ECEIZALAP /home/etor/work/freva/github/freva] (esgf-docu)
$ python --version
Python 3.11.4

```

I updated the enviornment:

```
mamba env update -f dev-environment.yml 
```

I modified the `make lint` so it would run linters from 3.7 to 3.11 and there is nothing

UPDATE:

I did `make install`
and now I am getting the errors

eelucio commented:
Those are very old code snippets. I will work on their var annotation later in the week
